source/common/grpc/common.h

```diff
@@ -28,12 +28,47 @@ class Common {
   /**
    * Charge a success/failure stat to a cluster/service/method.
    * @param cluster supplies the target cluster.
+   * @param protocol supplies the downstream protocol in use, either gRPC or gRPC-Web.
```

Comment:
Done by enumerate the actual valid values.
I actually prefer the http1 bridge use "http1bridge" instead of "grpc", to track the downstream protocol.
It should be:
"grpc": for grpc to grpc proxying.
"grpc-web": for grpc-web to grpc proxying.
"http1bridge": for http1bridge to grpc proxying.
"http-rest": for http/rest to grpc proxying.
However, as lyft has reporting tools relies on the http1bridge stats, I didn't change it.
@mattklein123 probably consider to change in the future?